src/partials/footer.html

```diff
@@ -0,0 +1,115 @@
+            </p>
+            <p class="contacts-workhours">
+              Saturday - Sunday <time class="contacts-workhours-wrapper">08:00 AM - 16:00 PM</time>
+            </p>
+            <hr>
+            <!--nie do końca jestem tego pewien znacznik (TIME)-->
+            <a href="tel:+61(0) 383 766 284" class="contacts-tel">+61(0) 383 766 284</a>
+            <a href="mailto:noreply@envato.com" class="contacts-email">noreply@envato.com</a>
+          </li>
+          <li class="contacts-main-item">
+            <p class="contacts-place-type">FOODTRUCK</p>
+            <p class="contacts-city">
+              Los Angeles
+              <a href="#" class="contacts-adress"
+                >1003 Manhattan Ave, Manhattan Beach, CA 90263, USA</a
+              >
+              <hr>
+              <!--link do gmaps w #-->
+            </p>
+            <p class="contacts-workhours">
+              Monday - Friday <time class="contacts-workhours-wrapper">06:00 AM - 10:00 PM</time>
+            </p>
+            <p class="contacts-workhours">
+              Saturday - Sunday <time class="contacts-workhours-wrapper">08:00 AM - 16:00 PM</time>
+            </p>
+            <hr>
+            <!--nie do końca jestem tego pewien znacznik (TIME)-->
+            <a href="tel:+61(0) 383 766 284" class="contacts-tel">+61(0) 383 766 284</a>
+            <a href="mailto:noreply@envato.com" class="contacts-email">noreply@envato.com</a>
+          </li>
+          <li class="contacts-main-item">
+            <p class="contacts-place-type">CAFE</p>
+            <p class="contacts-city">
+              New York
+              <a href="#" class="contacts-adress">726 Sterling Pl, Brooklyn, NY 11222, USA</a>
+              <hr>
+              <!--link do gmaps w #-->
+            </p>
+            <p class="contacts-workhours">
+              Monday - Friday <time class="contacts-workhours-wrapper">06:00 AM - 10:00 PM</time>
+            </p>
+            <p class="contacts-workhours">
+              Saturday - Sunday <time class="contacts-workhours-wrapper">08:00 AM - 16:00 PM</time>
+            </p>
+            <hr>
+            <!--nie do końca jestem tego pewien znacznik (TIME)-->
+            <a href="tel:+61(0) 383 766 284" class="contacts-tel">+61(0) 383 766 284</a>
+            <a href="mailto:noreply@envato.com" class="contacts-email">noreply@envato.com</a>
+          </li>
+        </ul>
+        <ul class="contacts-buttons">
+          <li class="contacts-buttons-locations">
+            <button type="button" class="contacts-button-location-wrapper">Our Location</button>
+          </li>
+          <!--dodac otwiertanie modala na lokalizacje-->
+
+          <li class="contacts-button-franchise">
+            <button type="button" class="contacts-button-franchise-wrapper">Franchise</button>
+          </li>
+          <!--dodac otwiertanie modala na franczyze-->
+        </ul>
+      </section>
+    </main>
+
+    <footer>
```

Comment:
Wszystkie super, chciałbym tylko żeby zachować spójność, by każda sekcja była oddzielnie, wiec prosiłbym to robić na dwa pliki. Contacts i footer :D